flixel/system/FlxSound.hx

```diff
@@ -94,12 +94,10 @@ class FlxSound extends FlxBasic
 	 */
 	public var volume(get, set):Float;
 
-	#if (sys && openfl_legacy)
```

Comment:
We may to add a new FlxDefine for FLX_PITCH, defined as `#if ((sys && openfl_legacy) || (lime > 3.10.0 && !flash))` or whatever version of lime is released next

Reply:
We could but all the code that's doing isn't trying to grab anything from the dev lime, just grabbing internal shit that the latest released lime version should work with

Reply:
what I mean is, we should use #if directives to hide the sound.pitch field entirely on targets that do not support it rather than exposing it when it doesn't work.

for instance, if someone is on flash `mySound.pitch` should give an error.

In this case I recommend creating the compiler directive FLX_SOUND_PITCH which behave similarly to `#if ((sys && openfl_legacy) || (lime > 3.10.0 && !flash))`, this way if we make features that use pitch, or if devs want to conditionally use the pitch feature they have an easy way to do so.

I'm not sure if openfl_legacy or lime 3.10 are still in use, so it may just be `#if !flash`, but in general it's good practice to use a specific compiler directive just in case this feature breaks on a certain target, or gets added to flash, or whatever

Reply:
I believe this is resolved in some way